src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l $padding-page;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: 24px;
```

Comment:
There is an existing var in `vars` that represents `24px`, but it's called `$font-size-form-header`. I originally used that, but then thought to myself "hey, self, this isn't a form header". Do you think I should just use `$font-size-form-header` here, or create a new var with a more generic name, like `$font-size-m-l`? 

Or maybe I should just use the existing `$font-size-l`, which is `28px` because that will make it more consistent with the rest of the site? `24px` is coming from the Sketch file, and I think one issue we might be hitting with these hero shelves is that the design specs have changed for Trailhead, so maybe we *shouldn't* be trying to be consistent with the rest of the site??

Reply:
For this initial launch I feel that we should try to keep the design consistent with the current AMO site. If you can use the existing variable for `28px` then that would be good. This type of subtle consistency is really important IMO. When we make all of this more trailhead-compliant we can hopefully address all of the historic inconsistencies that have crept in.